kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ReplaceQuestions.vue

```diff
@@ -78,12 +78,12 @@
               :style="{ userSelect: dragActive ? 'none!important' : 'text' }"
             >
               <ContentRenderer
-                :ref="`contentRenderer-${question.id}`"
```

Comment:
It might still be necessary to have this be `id` rather than `question_id` to prevent  a ref collision?